src/main.ts

```diff
@@ -324,6 +325,9 @@ export class Purchases {
 
   /** @internal */
   private async fetchAndCacheBrandingInfo(): Promise<void> {
+    if (isRCTestStoreApiKey(this._API_KEY)) {
+      throw Error("Branding info is not available for RC Test Store API keys.");
```

Comment:
This will be expected when using a Test Store API Key. Should we log a **warning** in this case?

Reply:
Yeah good point, I think a warning makes more sense 👍